recipes/quandl/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "Quandl" %}
+{% set version = "3.3.0" %}
+{% set sha256 = "d0775db98e4d0f8e031825df896abda7260ab5169ff2cfadd16bb0ad32f08111" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - pandas >=0.14
+    - numpy >=1.8
+    - requests >=2.7.0
+    - inflection >=0.3.1
+    - python-dateutil
+    - six
+    - more-itertools
+    - pyopenssl  # [py27]
+    - ndg-httpsclient  # [py27]
+    - pyasn1  # [py27]
+
+test:
+  imports:
+    - test
+
+about:
+  home: https://www.quandl.com/
+  license: MIT
+  license_family: MIT
+  # have requested to add it to the upstream
+  # license_file: LICENSE.txt
```

Comment:
Can you please package the license?

Reply:
There is no license in the upstream; it is not even in the github source. Can you please suggest me what to do?

Reply:
Then you are good to go I guess. (although you could include a link to the issue asking for a license)

Reply:
Let me do that and then if you think we are ready you can merge ;)